modules/cnf-installing-sriov-fec-operator.adoc

```diff
@@ -0,0 +1,173 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="installing-the-sriov-fec-operator_{context}"]
+= Installing the SR-IOV FEC operator
+
+The role of the OpenNESS Operator for Intel Wireless Forward Error Correction (FEC) Accelerator is to orchestrate and manage the devices exposed by a range of Intel's vRAN FEC acceleration hardware within the OpenShift cluster.
+
+One of the most compute-intensive 4G/LTE and 5G workloads is RAN layer 1 (L1) forward error correction (FEC), which resolves data transmission errors over unreliable or noisy communication channels. FEC technology detects and corrects a limited number of errors in 4G/LTE or 5G data without the need for retransmission.
+
+The FEC devices are provided by the Intel® FPGA PAC N3000 FPGA and the Intel® vRAN Dedicated Accelerator ACC100 for the vRAN use-case.
+
+[NOTE]
+====
+The Intel® FPGA PAC N3000 FPGA requires flashing with an 4G/LTE or 5G bitstream.
+====
+
+
+The SR-IOV FEC operator provides functionality to create desired virtual functions (VFs) for the FEC device, binds them to appropriate drivers, and configures the VFs queues for desired functionality in 4G/LTE or 5G deployment.
+
+As a cluster administrator, you can install the OpenNESS SRIOV-FEC Operator using the {product-title} CLI or the web console.
+
+[id="installing-sriov-fec-operator-using-the-cli_{context}"]
+== Installing the SR-IOV FEC Operator using the CLI
+
+As a cluster administrator, you can install the Operator using the CLI.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware.
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the SRIOV-FEC Operator by completing the following actions:
+
+.. Create the following Namespace Custom Resource (CR) that defines the `vran-acceleration-operators` namespace,
+and then save the YAML in the `sriov-namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: vran-acceleration-operators
+    labels:
+       openshift.io/cluster-monitoring: "true"
+----
+
+.. Create the namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-namespace.yaml
```

Comment:
Unsolicited feedback...any reason not to use `oc apply` since IIUC based on a recent comment from Vikram in forum-docs, that we have been recommending the use of `oc apply` since forever.

Reply:
I am following along on this guidance here https://docs.openshift.com/container-platform/4.7/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#installing-the-performance-addon-operator_cnf-master and modelled this section on that section. I did read your discussion on oc apply. I meant to bring it up at the team meeting but from our docs, there seems to be a lot of `oc apply` instructions. So I followed the same when it comes to applying the CR. Here I am creating the `Namespace` and `OperatorGroup` CR  later I come to apply the CR that defines the test and I use `oc apply`.

Reply:
@mikemckiernan, this is basically the same pattern for Operator installation used everywhere. I've updated the SR-IOV Network Operator variant to use here documents and so forth, but it never caught on I guess.